tests/functional/spec/features/validation/sum/value_source.spec.js

```diff
@@ -51,7 +52,7 @@ describe("Feature: Sum of grouped answers equal to validation against value sour
 
       await answerBothBreakdownQuestions(["3", "3", "3", "3"], ["2", "2", "1", "1"]);
 
-      await expect(browser).toHaveUrlContaining(SubmitPage.pageName);
+      await expect(browser).toHaveUrlContaining(AnotherTotalPlaybackPage.pageName);
```

Comment:
The test is a `Then I should get to the summary page` which doesn't align, probably needs to be "then i get to the next calculated summary" or continue once more to the submit page